### PR TITLE
feat(ci): semi-automate code-server-aur update

### LINKED ISSUE #5366: [Chore]: open PR into code-server-aur on publish
## What is your suggestion?

On publish after npm package, open PR to update https://github.com/coder/code-server-aur

## Why do you want this feature?

To save us some time and make it easier to update code-server on AUR

## Are there any workarounds to get this functionality today?

Manually doing it. 

## Are you interested in submitting a PR for this?

Yes

Related:
- https://github.com/coder/code-server/issues/2845

### PR DESCRIPTION
**Pull request by jsjoeio**
## Description 

This automates part of the release process for updating [code-server AUR package](https://aur.archlinux.org/packages/code-server). After the npm package is published, it will run this job which will update some files and open a PR to code-server-aur.

### Changes
- add new `aur` job to npm/brew workflow which runs on releases

### Testing

I was able to test this by adding a temporary job to `ci.yaml` and it successfully [opened a PR](https://github.com/coder/code-server-aur/pull/12).

image

### Next Steps
- [x] open issue to automate remaining steps
https://github.com/coder/code-server-aur/issues/13

Fixes #5366

### CODE REVIEW

File: .github/workflows/npm-brew.yaml [hunk 1]
```diff
@@ -67,3 +67,52 @@ jobs:
         env:
           HOMEBREW_GITHUB_API_TOKEN: ${{secrets.HOMEBREW_GITHUB_API_TOKEN}}
         run: ./ci/steps/brew-bump.sh
+
+  aur:
+    needs: npm
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    env:
+      GH_TOKEN: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
```
Comment (jsjoeio) [inline .github/workflows/npm-brew.yaml:76]: I feel like we should rename the secret to `CDRCI_GITHUB_API_TOKEN` if we're going to use it in both places. I'll have to find the value or ask @deansheather again though if I do.

Comment (code-asher) [reply .github/workflows/npm-brew.yaml:76]: Makes sense.  For now this seems chill.

**Review by code-asher — APPROVED** [review]
We might want to rename the workflow since it only mentions NPM and Brew.  Maybe we can simply call it `publish.yaml` and even move the Docker push here.

**Comment by jsjoeio** [comment]
> We might want to rename the workflow since it only mentions NPM and Brew. Maybe we can simply call it publish.yaml and even move the Docker push here.

Good idea! I'll do that in a followup PR.

**Pull request merged**